examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1189,16 +1313,34 @@ def compute_embeddings(prompt, text_encoders, tokenizers):
         unet = unet.to(torch.float32)
```

Comment:
Why do we have a `.to(torch.float32)` here actually? I don't think we need this no? 
```suggestion
```

Reply:
We always serialize in full precision unless specified otherwise.